### DISCUSSION TITLE
Being Oauth2 Provider, but can sign in with google

### DISCUSSION THREAD

**Discussion by muhajirdev** [ory/hydra#3297] [Q&A]
Hey, so we already have sign in with google and facebook. 

Now we are thinking to open the access to our user’s data to third party developer. Hence we need to implement the oauth2. 

But is it a good idea to do nested oauth2 like this? I mean user click “Continue with x” after that they being presented with a screen. Asking them to continue with google. Again.

Is there any security concern?

**Comment by vinckr** [comment]
Authentication (OIDC "login with google") and Authorization (OAuth2 consent flow) are two different areas, so IMO should be fine.
